server/controllers/assets/utils/determineRequestType.js

```diff
@@ -0,0 +1,61 @@
+const logger = require('winston');
+const { EMBED, BROWSER, SOCIAL } = require('../constants/request_types.js');
+
+function headersMatchesSocialBotList (headers) {
```

Comment:
This could be simplified by using an object instead of an array
```
const socialBotList = {
    'facebookexternalhit': 1,
    'Twitterbot': 1,
}
​
if (socialBotList[userAgent]) {
  logger.debug();
  return true;
} else {
  return false;
}
```

Reply:
cool pattern.  I updated to use this.

Reply:
@seanyesmunt do you prefer that to just:
```
const haystack = ['a', 'b']
if (haystack.includes(needle))
{
  ...
}
```

Reply:
(_technically_ `includes` is presumably `O(n)` and a map lookup `O(1)`, but this is such an efficient operation that this  doesn't matter)